pkg/machine/e2e/init_windows_test.go

```diff
@@ -2,14 +2,18 @@ package e2e_test
 
 import (
 	"fmt"
+	"os"
+	"os/exec"
 	"path/filepath"
 
 	"github.com/Microsoft/go-winio/vhd"
 	"github.com/containers/libhvee/pkg/hypervctl"
 	"github.com/containers/podman/v5/pkg/machine/define"
+	"github.com/containers/podman/v5/pkg/machine/wsl/wutil"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	. "github.com/onsi/gomega/gexec"
+	"github.com/twitchyliquid64/golang-asm/obj/wasm"
```

Comment:
this looks wrong

Reply:
wow i have no idea where that came from will fix